src/stories/Button.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Story, Meta } from '@storybook/react';
+
+import Button  from '../components/Button.component';
+import {ButtonProps} from "../components/Button.types"
+
+export default {
+    title: 'CommunityUI/Button',
+    component: Button,
+    argTypes: {
+    },
+} as Meta<typeof Button>;
+
+const Template: Story<ButtonProps> = (args) => <Button {...args} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+    primary: true,
+    disabled: false,
+    text: 'Primary',
```

Comment:
We should stick with Double quotes or single quotes throughout the whole project..I prefer single quotes but I think we should change to make it more consistent in all files.

Reply:
Yea, copy pasta my friend. 
We should probably create a style guidelines sheet and outline this in the CONTRIBUTING.md

Also, I like single quotes as well. 

You've already got some experience as an engineer, I bet. Good catch.

Reply:
Perhaps we can add a Prettier config file for consistent formatting.